### PR TITLE
Improve ARM64 support of the `sdk` function

### PR DESCRIPTION
**Pull request by dscho**
In Git for Windows' SDK, we [prominently advertise the `sdk` function](https://github.com/git-for-windows/build-extra/blob/53403f702e139577941baf9dda4f5308b20e09ce/git-extra/git-sdk.sh#L62-L63), but it does not _quite_ Do The Right Thing in `git-sdk-arm64` yet. This PR addresses that.

### CODE REVIEW

File: git-extra/git-sdk.sh [hunk 1]
```diff
@@ -72,12 +72,27 @@ sdk () {
...
+		MINGW32) shortcut_suffix=" 32-bit";;
+		MINGW64) shortcut_suffix=" 64-bit";;
+		CLANGARM64) shortcut_suffix=" ARM64";;
+		MSYS)
+			case "$(uname -m)" in
+			i686) shortcut_suffix=" 32-bit";;
+			x86_64)
+				if test -d /clangarm64
+				then
+					shortcut_suffix=" ARM64"
```
Comment (dennisameling) [inline git-extra/git-sdk.sh:85]: Do we want to make this lowercase `arm64` as discussed [here](https://github.com/git-for-windows/build-extra/pull/452#discussion_r1053792396)?

**Review by dennisameling — APPROVED** [review]
LGTM - left one minor comment. Thanks!

File: git-extra/git-sdk.sh (hunk 1 above)
Comment (dscho) [reply git-extra/git-sdk.sh:85]: That's a really good question. I am quite at a loss what we should do here. Maybe you can come up with sensible guidelines when to use upper-case "ARM64" and when to use lower-case "arm64" and when to use "clangarm64" and when to use "aarch64"?

Comment (dennisameling) [reply git-extra/git-sdk.sh:85]: Will create a discussion in `git-for-windows/git` for that later today 👍🏼

Comment (dennisameling) [reply git-extra/git-sdk.sh:85]: Slightly delayed, but here's the discussion: https://github.com/git-for-windows/git/discussions/4203

Comment (dennisameling) [reply git-extra/git-sdk.sh:85]: > That's a really good question. I am quite at a loss what we should do here. Maybe you can come up with sensible guidelines when to use upper-case "ARM64" and when to use lower-case "arm64" and when to use "clangarm64" and when to use "aarch64"?

I just added some guidelines to the [architecture discussion](https://github.com/git-for-windows/git/discussions/4203). It basically means that the artifact suffix becomes `arm64` (lowercase). How does that sound? 😊

Comment (dscho) [reply git-extra/git-sdk.sh]: > It basically means that the artifact suffix becomes `arm64` (lowercase). How does that sound? 😊

Sounds good! I did exactly that and force-pushed.

**Comment by dscho** [comment]
> /deploy mingw-w64-git-extra

That didn't work because the directory is still named `git-extra`.

> /deploy git-extra

That worked!

**Pull request merged**